Append a chaos version number.

Index: ldiskfsprogs/version.h
===================================================================
--- ldiskfsprogs.orig/version.h	2012-04-19 17:11:19.000000000 -0700
+++ ldiskfsprogs/version.h	2012-04-19 17:12:38.069236493 -0700
@@ -7,5 +7,5 @@
  * file may be redistributed under the GNU Public License v2.
  */
 
-#define E2FSPROGS_VERSION "1.41.90.wc4"
+#define E2FSPROGS_VERSION "1.41.90.4chaos.wc4"
 #define E2FSPROGS_DATE "01-Sep-2011"
Index: ldiskfsprogs/e2fsprogs-RHEL-6.spec.in
===================================================================
--- ldiskfsprogs.orig/e2fsprogs-RHEL-6.spec.in	2012-04-19 17:11:20.000000000 -0700
+++ ldiskfsprogs/e2fsprogs-RHEL-6.spec.in	2012-04-19 17:11:35.368445874 -0700
@@ -4,7 +4,7 @@
 Summary: Utilities for managing ext2, ext3, and ext4 filesystems
 Name: @E2FSPROGS@
 Version: @E2FSPROGS_PKGVER@
-Release: 7%{?dist}
+Release: 0%{?dist}
 
 # License tags based on COPYING file distinctions for various components
 License: GPLv2
